tests/sentry/incidents/endpoints/test_organization_alert_rule_details.py

```diff
@@ -886,8 +894,10 @@ def test_anomaly_detection_alert_update_other_error(self, mock_seer_request):
             status_code=400,
             **data,
         )
-        assert resp.data["detail"]["message"] == "Invalid request"
-        assert mock_seer_request.call_count == 2
+        assert resp.data[0] == ErrorDetail(
```

Comment:
@mifu67 will this mess up anything you did for the error responses showing up correctly?